### ISSUE TITLE
[A11y] Backend: Review Submissions

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3357]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Reference link: https://oregondigital.org/admin/workflows?locale=en

Logged in sidebar Tasks > Review Submissions

Ran the page through Accessibility Insights and recieved the following errors:

<img width="927" alt="Image" src="https://github.com/user-attachments/assets/27cc206a-b4d7-42a7-b069-e7a4205301ef" />

### Suggested Fix

Fix any of the following:

Element has insufficient color contrast of 4.04 (foreground color: #757575, background color: #f0f0f0, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Revalidate the page using Accessibility Insights.
